File: apps/home-service-platform-node-js-admin/src/api/serviceCategory/ServiceCategory.ts
import { Booking } from "../booking/Booking";
import { ServiceProvider } from "../serviceProvider/ServiceProvider";

export type ServiceCategory = {
  bookings?: Array<Booking>;
  categoryName: string | null;
  createdAt: Date;
  description: string | null;
  id: string;
  serviceProvider?: ServiceProvider | null;
  updatedAt: Date;
};

File: apps/home-service-platform-server/src/APIs/ServiceCategory/Dtos/ServiceCategory.cs
namespace HomeServicePlatform.APIs.Dtos;

public class ServiceCategory
{
    public List<string>? Bookings { get; set; }

    public string? CategoryName { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? Description { get; set; }

    public string Id { get; set; }

    public string? ServiceProvider { get; set; }

    public DateTime UpdatedAt { get; set; }
}
